v8/rest/template.ts

```diff
@@ -0,0 +1,33 @@
+import type { APIGuild, APITemplate } from '../payloads';
+
+/**
+ * https://github.com/discord/discord-api-docs/blob/master/docs/resources/Template.md#get-template--get-guildstemplatestemplatecodedocs_resources_templatetemplate-object
+ */
+export type RESTGetAPITemplateResult = APITemplate;
+
+/**
+ * https://github.com/discord/discord-api-docs/blob/master/docs/resources/Template.md#create-guild-from-template--post-guildstemplatestemplatecodedocs_resources_templatetemplate-object
+ */
+export interface RESTPostAPITemplateJSONBody {
```

Comment:
So this is what... create guild with template? If so, this should..really be renamed to make it clearer 🙏  (something like `RESTPostAPIGuildCreateWithTemplate` or something.. it's an edge case where name doesn't match route